frontend/src/components/common/input/input.module.scss

```
@@ -42,6 +42,8 @@
 }
 
 .error {
+  margin-top: 8px;
+  margin-bottom: 12px;
```

```suggestion
.input {
  margin: 0 0 5px;
```
common component should know anything about spaces between them. add spaces on the `form` level

---

don't forget about textaria

---

I think distance between input and its error message is too large. We need to reduce this distance